### ISSUE TITLE
Maya plugin cannot handle transformed pivots?

### ISSUE THREAD

**Issue by AWhetter** [Exocortex/ExocortexCrate#45]
Maybe it's just us but when we attach an Alembic to some existing geometry, we find that unless the pivot points for the geometry are at the origin, the transforms applied from the alembic are incorrect. We've tried a range of different export options but none have seemed to help.
We're working around anyway but it would be nice to not have to change the pivots of geometry (eg for the purpose of attaching lights).
The parameters we're using for exporting at the moment are
```
"normals=0",
"uvs=0",
"facesets=0",
"useinitshadgrp=0",
"dynamictopology=0",
"transformcache=0",
"globalspace=0",
"ogawa=1"
```

**Comment by bhouston** [comment]
I do not believe we support more than just a 4x4 transform per node.  It may be that we are not taking into account pivot points when creating that 4x4 transform here.
